### ISSUE TITLE
Mouse Buttons and Wheel

### ISSUE THREAD

**Issue by MadokaAyukawa** [wangqr/Aegisub#5]
Currently Aegisub doesn't seem aware of the extra buttons on a five button mouse. I would like to be able to assign the extra mouse buttons to hotkeys. For example, if I could assign Mouse4 and Mouse5 to "Line Back" (time/prev) and "Commit", I could do basic timing with the mouse alone.

With the audio window active, the mouse wheel will scroll the audio window. But one click of the wheel scrolls the window much more than "audio/scroll/right". It would be nice of one could adjust the mouse wheel sensitivity to control the amount one click will scroll the audio window.

**Comment by wangqr** [comment]
> Currently Aegisub doesn't seem aware of the extra buttons on a five button mouse.

Yes, the additional keys are usually used as navigation back / forward. I could not come up with a intuitive function for these buttons. Sure it can be assigned to some function, but that needs new mouse event handling code. As I don't have 5 button mouse, I will not work on this in near future.
I believe there are drivers / 3rd party tools that can map those buttons to keyboard shortcuts, which can then be customized to function you need. Feel free to create PR if you want Aegisub support it natively.

> But one click of the wheel scrolls the window much more than "audio/scroll/right".

Scrolling one increment (tick) using mouse wheel is set to scroll 1/3 of the displayed width. I'll change that to absolute distance, which feels more natural to me.

**Comment by MadokaAyukawa** [comment]
Thank you, as I felt 1/3 the display width was too far as well as it would always go past my intended target.

**Comment by MadokaAyukawa** [comment]
The feel and scroll of the wheel is much better now. Thank you.
